python/ctsm/crop_calendars/check_rxboth_run.py

```diff
@@ -128,22 +136,27 @@ def main(argv):
 
         # Check
         if case["rx_sdates_file"]:
-            check_rx_obeyed(
+            sdate_not_obeyed = check_rx_obeyed(
                 case["ds"].vegtype_str.values,
                 case["rx_sdates_ds"].isel(time=0),
                 case["ds"],
                 casename,
                 "SDATES",
             )
+            any_bad = any_bad or sdate_not_obeyed
         if case["rx_gdds_file"]:
-            check_rx_obeyed(
+            gdds_not_obeyed = check_rx_obeyed(
                 case["ds"].vegtype_str.values,
                 case["rx_gdds_ds"].isel(time=0),
                 case["ds"],
                 casename,
                 "GDDHARV",
                 gdd_min=gdd_min,
             )
+            any_bad = any_bad or gdds_not_obeyed
+
+    if any_bad:
+        raise RuntimeError("Unexpected behavior in rxboth run")
```

Comment:
Error message should say what exactly failed (if printed messages haven't already said so).

Reply:
Done with b96ec9107.